### PR TITLE
Document the snapshot export to S3

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Document the snapshot export to S3

Documents #486 

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Reviewer
- [x] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: docs/modules/hbase/pages/usage-guide/snapshot-export.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+= Exporting a snapshot to S3
+
+HBase snapshots can be exported with the command `hbase snapshot export`.
+To be able to export to S3, the AWS libraries from Hadoop must be on the classpath.
+These libraries are contained in the HBase image at `/stackable/hadoop/share/hadoop/tools/lib/`.
+The script `export-snapshot-to-s3` facilitates the export, by providing the according classpath, extending the Hadoop configuration with the S3 configuration, and calling `hbase snapshot export`.
```
Comment (adwk67) [inline docs/modules/hbase/pages/usage-guide/snapshot-export.adoc:6]:
```suggestion
The script `export-snapshot-to-s3` facilitates the export, by providing the associated classpath, extending the Hadoop configuration with the S3 configuration, and calling `hbase snapshot export`.
```

File: docs/modules/hbase/pages/usage-guide/snapshot-export.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    --snapshot snap \
+    --copy-from s3a://my-bucket/my-snapshot \
+    --copy-to hdfs://simple-hdfs/hbase
+```
+
+However, the better approach is to create a Job:
+
+[source,yaml]
+----
+include::example$usage-guide/snapshot-export-job.yaml[]
```
Comment (adwk67) [inline docs/modules/hbase/pages/usage-guide/snapshot-export.adoc:51]: I think we have to split the script here, as at least in my IDE it only renders the last part (the secret).

**Review by adwk67 — CHANGES_REQUESTED** [review]

File: docs/modules/hbase/pages/usage-guide/snapshot-export.adoc (hunk 2 above)
Comment (siegfriedweber) [reply docs/modules/hbase/pages/usage-guide/snapshot-export.adoc:51]: I cannot reproduce this issue with Antora. Nevertheless, I put the YAML documents into separate files. Please try again, if it works now in your IDE.

Comment (adwk67) [reply docs/modules/hbase/pages/usage-guide/snapshot-export.adoc:51]: Yes, this works for me now in my IDE (Rust Rover). Thanks!

**Review by adwk67 — APPROVED** [review]

**Comment by lfrancke** [comment]
Can you please include a link to the generated nightly docs?

**Comment by fhennig** [comment]
https://docs.stackable.tech/home/nightly/hbase/usage-guide/snapshot-export

**Pull request merged**